src/test/java/com/epam/ta/reportportal/ws/utils/SingletonObjectMapper.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *  * Copyright 2023 EPAM Systems
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.epam.ta.reportportal.ws.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+public class SingletonObjectMapper {
+
+  private static ObjectMapper instance = null;
+
+  public static ObjectMapper getInstance() {
+    if (instance == null) {
+      ObjectMapper mapper = new ObjectMapper();
+      mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```

Comment:
WRITE_DATES_AS_TIMESTAMPS should be used in every module in order to serialize dates properly